SixTest/check_stf

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+# this script is exact copy of check_90 
+# with the difference that it will
+# compare fort.90 from sixtrack stf version with 
+# fort.90.stf_canonical in test folders
+
+prov=canonical
+echo "------------------------------------------------------------"
+echo "This is for comparing stf version of sixtrack with canonical"
+echo "If you haven't chosen single track file flag option then    "
+echo "run check_90 instead                               "
+echo "------------------------------------------------------------"
+# Now uses the variable TESTS
+export BIN=`pwd`/bin
+if test $# -eq 0
+then
+  echo "You need to give a Version name (or two)!!!"
+  exit
+fi
+newv=$1
+if test $# -eq 2
+then
+  prov=$1
+  newv=$2
+fi
+# Now try and find EXECS to compare
+rm -rf etmp$$
+touch etmp$$
+for e in $TESTS
+do
+  if test ! -d "$e"
+  then
+    echo "      ===================================="
+    echo "      CASE $e directory does not exist!!!"
+    echo "      ===================================="
+  else
+    cd $e
+    ls singletrackfile.dat.*${newv}* >> ../etmp$$ 2>/dev/null
+    cd ..
+  fi
+done
+sort --unique etmp$$ > etmp2$$
+newvs=`cat etmp2$$ | sed -e's/singletrackfile.dat.//'`
+rm -rf etmp*
+for v in $newvs
+do
+  echo "++++++++++++++++++++++++++++++++++++++"
+  echo "Comparing $prov to $v"
+  echo "++++++++++++++++++++++++++++++++++++++"
+  tests=`echo "$TESTS" | sed -e's/ *bnl */ /'`
+  echo $tests
+  for i in $tests
+  do
+    if test ! -d "$i"
+    then
+      echo "      ===================================="
+      echo "      CASE $i directory does not exist!!!"
+      echo "      ===================================="
+    else
+      echo "      ===================================="
+      echo "      CASE $i"
+      echo "      ===================================="
+      if test -s $i/singletrackfile.dat.$prov
+      then 
+          if test -s $i/singletrackfile.dat.$v
+          then
+              mkdir -p tmp$$
+              cd tmp$$
+              cp ../$i/singletrackfile.dat.$prov fort.190
+              $BIN/read90 > fort.20 
```

Comment:
Hmm, there should probably be a special version of read90 which reads all of the new files...